source/draft_api.rst

```diff
@@ -186,19 +186,16 @@ Response body:
     {
       "duration":12,
       "user": "example-2",
-      "project": "",
+      "project": "qa",
```

Comment:
Project is sent in a POST request as a slug, but returned from a GET request as an array.